joomla/joomla-cms
# [4.0] Switcher is now a Custom Element

### Summary of Changes

This PR migrates the Switcher field over to a dedicated custom element which has been approved by the a11y team.

### Testing Instructions

Go to the Global Configuration (or any other page that uses a switcher) and ensure it works the same as it did before.

**_Note_**, _please ignore the colours_. We still need to publish an update to NPM and properly pull the elements using the new package manager. Once done and we start working on the new template, we'll work the the necessary SCSS changes.

Hope @brianteeman likes the removal or the border radius :)
